.env.example

```diff
@@ -1,27 +0,0 @@
-APP_ENV=local
```

Comment:
This file shouldn't have been deleted from Git. It's a template so anyone can set up their own `.env` when they first set up their local app.